pallets/grant/src/weights.rs

```diff
@@ -18,52 +18,32 @@
 //! Autogenerated weights for pallet_grant
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-07-31, STEPS: `100`, REPEAT: 50, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
Just FYI. We don't have to regenerate the weights every time. We just have to make sure that there are benchmarks for each extrinsic. Ideally, we generate all weights before the official release on some reference machine.

Reply:
Ok cool.